File: JAVA/StringDemo.java
class StringDemo 
{
    public static void main(String Google[])
    {
        String s1 = "Hello";
        String s2 = new String("Hello");
        
        char Arr[] = {'H','E','L','L','O'};
        String s3 = new String(Arr);

        System.out.println(s1);
        System.out.println(s2);
        System.out.println(s3);

        System.out.println(s1.length());
        System.out.println(s2.length());
        System.out.println(s3.length());
    }    
}

File: C/StringDemo.c
#include<stdio.h>
#include<string.h>
int main()
{
    char Arr[20];
    printf("Enter character");
    printf("%s", &Arr);
    printf("the string is %s", Arr);
    
    return 0;
}
/*
char Arr[] = "Hello";
    char Brr[] = {'H','E','L','L','O','\0'};

    printf("%s\n", Arr);
    printf("%s\n", Brr);

    printf("%d\n",strlen(Arr));
    printf("%d\n",strlen(Brr));


*/